src/main/java/com/pet/bankservice/repository/UserRepository.java

```diff
@@ -0,0 +1,18 @@
+package com.pet.bankservice.repository;
+
+import com.pet.bankservice.entity.User;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.stereotype.Repository;
+import java.util.Optional;
+
+@Repository
+public interface UserRepository extends JpaRepository<User, Long> {
+    @Query("FROM User u INNER JOIN FETCH u.roles WHERE u.id=?1")
+    Optional<User> get(Long aLong);
```

Comment:
Entity field is `id` so why it is `aLong` here?

Reply:
done